pallets/messages/src/lib.rs

```diff
@@ -376,7 +376,11 @@ impl<T: Config> Pallet<T> {
 		// insert into storage and create events
 		for (schema_id, messages) in map {
 			let count = messages.len() as u16;
-			let bounded_vec: BoundedVec<_, _> = messages.try_into().unwrap();
+			let bounded_vec: BoundedVec<_, _> = messages.try_into().unwrap_or_default();
+
+			if bounded_vec.is_empty() {
```

Comment:
I set this to the same return value as previous, because there were no additional db accesses at this point.